### ISSUE TITLE
crop causes color misalignment

### ISSUE THREAD

**Issue by cdgriffith** [rigaya/VCEEnc#30]
Example command: 

```
VCEEncC64.exe -i D:\BigBuckBunny.mp4 --crop 0,32,0,32 -o D:\BigMistake.mp4
```

Causes the output to look like: 

![bad](https://user-images.githubusercontent.com/3275435/125975165-b3a4db3a-6671-4eab-a7fd-e0f98d60bceb.jpg)

Where it should look like:

![good](https://user-images.githubusercontent.com/3275435/125975187-55798d26-ad7d-4b11-9195-1af78a0b3fb5.jpg)

It seems to somehow move the colors down?  Tested on multiple videos. Without the crop the colors look correct. 

Using drivers 21.5.2

```
VCEEncC64.exe --check-hw
VCEEncC (x64) 6.12 by rigaya, build May 23 2021 13:42:37
AMF Version 1.4.18
reader: raw, avs, vpy, avhw [H.264/AVC, H.265/HEVC, MPEG2, VP9, VC-1]
VCE available
device #0: AMD Radeon RX 6900 XT
```

**Comment by rigaya** [comment]
Thanks for reporting the issue, chroma handling was wrong for cropping.

Shall be fixed by [VCEEnc 6.13](https://github.com/rigaya/VCEEnc/releases/tag/6.13).

**Comment by cdgriffith** [comment]
Tested and working, thank you!
